### ISSUE TITLE
MVP definition

### ISSUE THREAD

**Issue by ferchault** [NablaChem/hummingqueue#3]
- fun python funcs
- cloudpickle

- fixed python envs: version and requirements

- decorator to convert args and kwargs

- group secret in file

- requirement compute nodes: conda

- functions in s3 storage, identified by artefact hash
- dashbaord: summary coming from mongodb 

- out of scope:
- large args
- non-hashable args
- multicore runs

only once:
hmq.setup("group secret")

@hmq.task
def test(a, b, c=4):
  return a+b+c

test() # run locally

test.remote([2,2], [3,3]) -> tag
tag.name = "test-timestamp-random"
tag.tasks = [UUID]

hmq.get_result(UUID)
hmq.get_input(UUID)
hmq.get_status(UUID)
hmq.get_status(tag)
hmq.clear(UUID) # cancel and delete input and output

Q: how to execute in particular env (exploit that one tag is one env)
